lib/spanner/command/reqrep.ex

```diff
@@ -57,12 +48,10 @@ defmodule Spanner.Command.Request do
 
   defp read_config(request, config_path) do
     [bundle, _cmd] = String.split(request.command, ":")
```

Comment:
Hrm, this has been here for a while and not been caught. I'll fix it and add a test.